src/ParallelAlgorithms/LinearSolver/ConjugateGradient/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_target_headers(
+  ParallelLinearSolver
+  INCLUDE_DIRECTORY ${CMAKE_SOURCE_DIR}/src
+  HEADERS
+  ConjugateGradient.hpp
+  ElementActions.hpp
+  InitializeElement.hpp
+  ResidualMonitorActions.hpp
```

Comment:
I think `.` is before `A` in the alphabet. (it was in `src/ParallelAlgorithms/EventsAndTriggers/CMakeLists.txt` in the previous commit... so one needs to be fixed)